container/dev/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghcr.io/stjude/ppbase:latest
+
+ARG IMGREV=""
+ARG IMGVER=""
+
+LABEL org.opencontainers.image.source="https://github.com/stjude/proteinpaint" \
+    org.opencontainers.image.revision="$IMGREV" \
+    org.opencontainers.image.version="$IMGVER"
+
+WORKDIR /home/root/proteinpaint/
+
+EXPOSE 3000
+CMD ["sh", "-c", "npm install && npm run build && npm run dev1"]
```

Comment:
Shouldn't `npm install && npm run build` already be done in a `RUN` step before `CMD`? Or is there a reason for running it as part of `CMD`?

Reply:
On second thought, maybe this is done in case someone installs a new npm dependency or updates an existing dep in a branch, and the majority of the node_modules can be reused on every docker start. But not 100% sure. Maybe can leave as-is and investigate for actual use in regular, day-to-day sjpp dev.

Reply:
The RUN command is used to execute commands during the image build process, and at that time the source is not attached yet. That's why I used CMD.